app/src/main/java/com/banuba/sdk/agorapluginexample/MainActivity.kt

```diff
@@ -49,19 +51,20 @@ class MainActivity : AppCompatActivity() {
             mContext = this@MainActivity
             mAppId = AGORA_APP_ID
             System.loadLibrary("banuba")
-            addExtension(BanubaExtensionManager.EXTENSION_NAME)
+            addExtension("banuba-plugin")
```

Comment:
Why you've changed this - the previous variant I liked more. At least define constant.